pkg/cli/crud.go

```diff
@@ -58,10 +58,11 @@ func getRemoveCommand() *cobra.Command {
 
 func getWatchCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "watch {device} [args]",
+		Use:   "watch {device|entity} [args]",
```

Comment:
should it be possible to watch for relationships too, or will they come together with entity?